pandas/core/frame.py

```diff
@@ -971,8 +971,6 @@ def iterrows(self) -> Iterable[Tuple[Label, Series]]:
         data : Series
             The data of the row as a Series.
 
-        it : generator
-            A generator that iterates over the rows of the frame.
 
```

Comment:
Could you please remove one of the empty rows? Double emtpy rows are not allowed

Reply:
Okay! Thx your review. I'll fix it.